wordpress.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	extends: ['plugin:@wordpress/eslint-plugin/custom', './react'],
+	globals: {
+		window: true,
+		document: true,
+		wp: 'readonly',
```

Comment:
@nicholasio Can we add the following?

```js
wp: true,
lodash: true
```

We use this on all projects

Reply:
`wp` is already there as readonly. Will add lodash.